database/factories.py

```diff
@@ -0,0 +1,73 @@
+import factory
+from .tableau_api_models import SummaryStatisticDaily
+from .user_models import Participant
+from .study_models import Study
```

Comment:
Let's use absolute instead of relative imports (`from database.tableau_api_models import...`)